src/index.js

```diff
@@ -36,34 +38,30 @@ export default class DesignSystem {
     return this.get(z, this.ds.zIndex)
   }
 
-  fontSize(size, toPxl = false) {
-    let output
-    if (this.options.useModularScale) {
-      const value =
-        typeof size === 'number' ? size : this.get(size, this.ds.type.sizes)
-      output = ms(value, this.ds.type.modularscale)
-    } else {
-      output = this.get(size, this.ds.type.sizes)
-    }
-
-    const untransformedOutput = `${output}px`
+  fontSize(size) {
+    const output = `${this.get(size, this.ds.type.sizes)}`
+    const baseFontSize = parseFloat(this.ds.type.baseFontSize)
 
-    if (toPxl) {
-      return untransformedOutput
+    // Don't convert the value if we don't have to
+    if (output.indexOf(this.options.fontSizeUnit) !== -1) {
```

Comment:
Wouldn't this return `true` if `this.options.fontSizeUnit` is set to `em`, but the output contains `rem`?

Reply:
@munkychop you are absolutely correct. I am working on a fix for this now.